### PR TITLE
Add section about issues to be translated

### PR DESCRIPTION
**Pull request by nixocio**
Add steps related to issues to be translated.

### CODE REVIEW

File: awx/ui_next/CONTRIBUTING.md
Comment (jlmitch5) [inline awx/ui_next/CONTRIBUTING.md:335]: do you mean component:I18n?

Comment (nixocio) [reply awx/ui_next/CONTRIBUTING.md:335]: @jlmitch5, I just followed the name of the label that we have on github.


image

File: awx/ui_next/CONTRIBUTING.md
Comment (rooftopcellist) [inline awx/ui_next/CONTRIBUTING.md]: It might be good to clarify step 3 by adding, "At this point, the translations will be batch translated by a maintainer, creating relevant entries in the PO files.  Then after those translations have been merged, the issue can be closed."

**Review by jlmitch5 — APPROVED** [review]
talked about the label yesterday...i was wrong that it needed to be something different.  looks good!

**Review by rooftopcellist — APPROVED** [review]

**Pull request merged**
